lib/improver/tests/test_nbhood_utilities.py

```diff
@@ -0,0 +1,290 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Unit tests for the nbhood.Utilities plugin."""
+
+
+import unittest
+
+from cf_units import Unit
+
+from iris.cube import Cube
+from iris.exceptions import CoordinateNotFoundError
+from iris.tests import IrisTest
+import numpy as np
+
+from improver.nbhood import Utilities
+from improver.tests.helper_functions_ensemble_calibration import (
+    add_forecast_reference_time_and_forecast_period)
+from improver.tests.test_nbhood_neighbourhoodprocessing import (
+    set_up_cube, set_up_cube_lat_long)
+
+
+class Test_cumulate_array(IrisTest):
+
+    """Test for cumulating an array vertically and horizontally."""
+
+    def test_basic(self):
+        """
+        Test that the vertical and horizontal accumulation produces the
+        intended result.
+        """
+        data = np.array([[5., 10., 14., 19., 24.],
```

Comment:
@arh89 @benfitzpatrick We haven't made any changes to this now, but this is useful information regarding the behaviour of the unit testing. I don't think that the docstrings for the unit tests have been standardised before, so thanks for raising this issue. This might feed into outstanding actions from Developer Catchups regarding docstring technical debt.